about.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <script src="https://cdn.tailwindcss.com"></script>
+    <title>Naledi's Capstone</title>
+  </head>
+
+  <body>
+    <header>
+      <ul
+        class="hidden items-center bg-gray-800 text-white md:flex justify-end h-[30px] pr-[10%]"
+      >
+        <li class="p-1">
+          <a href=""><i class="bi bi-facebook"></i></a>
+        </li>
+        <li class="p-1">
+          <a href=""><i class="bi bi-twitter"></i></a>
+        </li>
+        <li class="p-1"><a href="">English </a></li>
+        <li class="p-1"><a href="">My Page</a></li>
+        <li class="p-1"><a href="">logout</a></li>
+      </ul>
```

Comment:
Kindly give proper spacing to this section as in the provided template. I have recorded [this video](https://www.loom.com/share/995ea0b1c5eb4346a5f0d096bb8e3b5b) where you can verify the problem.